src/plugins/azure/ssh.ts

```diff
@@ -45,31 +54,74 @@ export const azureSshProvider: SshProvider<
 
   propagationTimeoutMs: PROPAGATION_TIMEOUT_LIMIT_MS,
 
-  // TODO: Implement
+  // TODO: Determine if necessary
```

Comment:
If sudo takes time to propagate we can use the preTestAccessPropagationArgs (see GCP) to verify that sudo works before actually opening the session for the user.
This is just fyi for later.

Reply:
Created https://linear.app/p0-security/issue/ENG-3149/ to track this work; will also update the TODO comment.